lib/disability_compensation/factories/api_provider_factory.rb

```diff
@@ -187,11 +187,17 @@ def generate_pdf_service_provider
   end
 
   def supplemental_document_upload_service_provider
+    provider_options = [
+      @options[:form526_submission],
+      @options[:document_type],
+      @options[:statsd_metric_prefix]
```

Comment:
idk if this was seen in the other one, but why are we passing statsd prefix around again? Why is this not just hardcoded in the provider to force all calls within a provider to have the same prefix?

It is not important enough to prevent merge and can always be changed later, I jsut forget if there was a reason for it to be like this

Reply:
There are a couple hardcoded statsd metrics that get chained together, this the one we I will pass from the job what will tell the provider what job is running it (e.g.  we already have a `worker.evss.submit_form526_bdd_instructions` metric for the BDD job), the providers themselves also have hardcoded metrics I'm including, so we know for that job, what provider we used, and what happened, so metrics we would build monitoring for would look like:

`worker.evss.submit_form526_bdd_instructions.lighthouse_supplemental_document_upload_provider.upload_attempt`

or `worker.evss.submit_form526_bdd_instructions.evss_supplemental_document_upload_provider.upload_job_failed` for example.


We could also pass our own prefix if we didn't want to use [the existing one](https://github.com/department-of-veterans-affairs/vets-api/blob/benefits-disability-nb-refactor-supplemental-document-upload-providers/app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb#L10), since `worker.evss.submit_form526_bdd_instructions` is kind of confusing if we are uploading via Lighthouse